docs/docsite/rst/user_guide/playbooks_advanced_syntax.rst

```diff
@@ -50,6 +50,50 @@ For complex variables such as hashes or arrays, ``!unsafe`` should be used on th
         unsafe_key: !unsafe 'unsafe value'
 
 
+Using nested variables
+~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to refer to the same object from different parts of playbook, you need to use anchors and aliases.
+Anchors are denoted by ``&`` indicator while aliases are denoted by ``*``
+
+In a playbook, this may look like::
+
+    ---
+    ...
+    vars:
+        app1:
+            jvm: &jvm_opts
+                opts: '-Xms1G -Xmx2G'
+                port: 1000
+                path: /usr/lib/app1
+        app2:
+            jvm:
+                <<: *jvm_opts
+                path: /usr/lib/app2
+    ...
+
+Here, 'app1' and 'app2' variables shares 'opts' and 'port' information so you can anchor that
+information using ``&jvm_opts`` and re-use this information by indicating ``<<: *jvm_opts``.
```

Comment:
I'm having difficulty following this, as someone not familiar with YAML's support for anchor and aliases.  Initially a * meant alias, if so, what's the <<: about here?

Also not sure how I would know that the alias only reuses 'opts' and 'port'. If I put 'port: 2000' for app2, that would mean it was only sharing 'opts'?  (that is, I can reuse part of a set of variables if I set values for the other variables directly where I'm reusing as the alias?)